### ISSUE TITLE
Legal history

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-bcase#58]
from [Minimize Legal Risk section](https://w3c.github.io/wai-bcase/teach-advocate/business-case/#minimize-legal-ris):

> Change is occuring incrementally and steadily.
> * Early 2000s in the US, a few legal actions related to inaccessible websites based on the Americans with Disabilities Act (ADA) and Section 508 of the Rehabilitation Act for US government entities.
> * Despite several encouraging victories in US courts for people with disabilities seeking equal access, Section 508 was seen to have limited scope and the ADA was unevenly applied by various courts.

I don't understand the importance of these points for a top-level page of a business case.

(This also contributes to the US-centric feel of the document overall.)

**Comment by yatil** [comment]
(Closed in error, should have closed #56)

**Comment by sharronrush** [comment]
Removed those two references, smoothed the text around it.
